Source/Pawnmorphs/Esoteria/MutationStagePatch.cs

```diff
@@ -110,7 +111,10 @@ private void Modify(MutationDef mutation)
                         // get default value
                         object defaultValue = member.GetValue(defaultValues);
 
-                        if (newValue.Equals(defaultValue))
+                        var collection = newValue as ICollection;
+                        if (collection != null && collection.Count < 1)
+                            continue;
```

Comment:
This could instad be written as 
if (newValue is ICollection collection && collection.Count == 0)
continue

That way you don't need to make a explicit variable or do a null check. I also don't see why you would do a less than 1 rather than equal 0. A collection can never have negative amount of entries, so using a less than (at least to me) indicates that your intention is to check for more than a single case.

Reply:
I was wondering how to do it in a single line.